src/base.ts

```diff
@@ -71,22 +74,69 @@ export default abstract class extends Command {
 
   got = got.extend()
   spinner!: ora.Ora;
-  readGlobalConfig(): IGlobalLiaraConfig {
-    let content
-
-    try {
-      content = JSON.parse(fs.readFileSync(GLOBAL_CONF_PATH).toString('utf-8')) || {}
-    } catch {
-      content = {}
+  async readGlobalConfig(): Promise<IGlobalLiaraConfig> {
+    if (fs.existsSync(GLOBAL_CONF_PATH)) {
+      fs.removeSync(PREVIOUS_GLOBAL_CONF_PATH);
+      const content = fs.readJSONSync(GLOBAL_CONF_PATH, {throws: false}) || {}
+      return content;
     }
-
-    // For backward compatibility with < 1.0.0 versions
-    if (content.api_token) {
-      content['api-token'] = content.api_token
-      delete content.api_token
+    const content = fs.readJSONSync(PREVIOUS_GLOBAL_CONF_PATH, {throws: false}) || {}
+    if (content.accounts && Object.keys(content.accounts).length) {
+      const accounts: IAccounts = {};
+      for (const account of Object.keys(content.accounts)) {
+        await this.setAxiosConfig({
+          "api-token": content.accounts[account].api_token,
+          region: content.accounts[account].region,
+        });
+        try {
+          const {
+            user: { email, fullname, avatar },
+          } = await got.get("v1/me").json<{ user: IAccount }>();
+
+          accounts[account] = {
+            email,
+            avatar,
+            fullname,
+            region: content.accounts[account].region,
+            api_token: content.accounts[account].api_token,
+            current: content.current === account ? true : false,
+          };
+        } catch (error) {}
```

Comment:
Please just ignore response errors, not all errors. Currently, we can't debug this section without seeing errors.